targets/ar71xx-mikrotik

```diff
@@ -1,9 +1,16 @@
 config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
 
+ATH10K_PACKAGES='kmod-ath10k-ct ath10k-firmware-qca9887 ath10k-firmware-qca988x'
+if [ "$GLUON_WLAN_MESH" = 'ibss' ]; then
+	ATH10K_PACKAGES='kmod-ath10k-ct ath10k-firmware-qca9887-ct ath10k-firmware-qca988x-ct'
+fi
```

Comment:
Adding packages without testing them is not helpful. If you're testing for Freifunk Bremen you've probably only tested IBSS. Please remember that support for 802.11s is a requirement and there have been issues with ath10k-ct and IBSS mesh (see #1584).

Reply:
Yeah. Right. IBSS does not work with ath10k concurrent with client network. I've changed the support only for 11s.